### DISCUSSION TITLE
Chrome dont send cookie with allow cors

### DISCUSSION THREAD

**Discussion by devopg** [axios/axios#5484] [Q&A]
i try to send POST request from localhost to another domain. I set for cookie `SameSite = None ; Secure`

how it require in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#see_also

![image](https://user-images.githubusercontent.com/61819835/213808945-ede04fae-3ba4-4428-9bdc-3327a6c23f03.png)

Request made to HTTPS server. Using AXIOS with

`{ withCredentials: true }`
But in request information i dont see that cookie send

![image](https://user-images.githubusercontent.com/61819835/213808973-40406172-e8f5-4b0b-a7d0-1b138c84301e.png)

What is problem? What did i miss?

On server side i use this:

```
app.use(
    cors({
        credentials: true,
        //origin: true,
    origin: function (origin, callback) {
      callback(null, true);
    },
    })
);
```
its give me in response headers:

![image](https://user-images.githubusercontent.com/61819835/213809002-2c4bf2aa-3c9a-4aa1-b776-7ae231b5d7f1.png)

**Comment by jasonsaayman** [comment]
This is controlled by the browser, not Axios. withCredentials: true only tells the browser it may include credentials. Chrome will still send the cookie only if the cookie’s Domain/Path match the request URL, it was accepted as SameSite=None; Secure, third-party cookies are not blocked, and the CORS response includes Access-Control-Allow-Credentials: true plus an explicit Access-Control-Allow-Origin matching your localhost origin. Also make sure you are calling the same HTTPS host that set the cookie.
